ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts

```diff
@@ -0,0 +1,41 @@
+import { GeneralCollateralIF, PartyIF, VehicleCollateralIF } from '@/interfaces'
+import { ErrorIF } from './error-interface'
+
+// Payment (pay-api) reference interface.
+export interface PaymentIF {
+  invoiceId: string, // Unique identifier of the payment transaction.
+  receipt: string // Payment API URI to obtain the payment receipt.
+}
+
+// Financing Statement interface. All dates/date time properties are in the ISO 8601 format YYYY-MM-DDThh:mm:ssTZD.
+export interface FinancingStatementIF {
+  type: string, // One of enum APIRegistrationTypes.
+  clientReferenceId?: string, // AKA folio max length 20.
+  documentId?: string, // Optional draft ID if draft created.
```

Comment:
I don't see anywhere where this documentId is set...

Reply:
I think all your type questions are answered by the looking at the API spec:
https://bcregistries-daxiom.web.app/ppr-api
ID's are generated by the API and returned in responses.